launcher/InstanceList.cpp

```diff
@@ -290,8 +290,7 @@ void InstanceList::deleteGroup(const GroupId& name)
             qDebug() << "Remove" << instID << "from group" << name;
             removed = true;
             auto idx = getInstIndex(instance.get());
-            if (idx > 0)
```

Comment:
it would be more corect to just change the condition from `idx>0` to `idx>=0` as the `getInstIndex` can return -1 as value.

Reply:
Ah... it shouldn't but yeah